pipeline/config/exoscale/wc-config.yaml

```
@@ -0,0 +1,370 @@
+    ## TODO: remove, we do not label our nodes with 'nodeType' by default.
+    ##
+    tolerations:
+      - key: nodeType
+        operator: Exists
+        effect: NoSchedule
+    affinity: {}
+    nodeSelector: {}
+    config:
+      ## If 'true', use PROXY protocol
+      ## ref: https://docs.nginx.com/nginx/admin-guide/load-balancer/using-proxy-protocol/
+      useProxyProtocol: false
+    ## If 'true', nginx will use host ports 80 and 443
+    useHostPort: true
+    ## Kubernetes service configuration.
+    service:
+      enabled: false
+      ## Type of service.
+      ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
+      type: set-me
+      ## Annotations to add to service
+      ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+      annotations: set-me
+    ## Additional configuration options for Nginx
+    ## ref: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/
+    additionalConfig: {}
+  defaultBackend:
+    resources: {}
+    ##
+    ## TDOD: remove, we do not label our nodes with 'nodeType' by default.
+    ##
+    tolerations:
+      - key: nodeType
+        operator: Equal
+        value: elastisys
+        effect: NoSchedule
+    affinity: {}
+    nodeSelector: {}
+## Configration for Velero and Restic.
+## Check out https://compliantkubernetes.io/user-guide/backup/ to see what's included in backups.
+velero:
+  enabled: true
+  tolerations: []
+  nodeSelector: {}
+  resources:
+    limits:
+      cpu: 200m
+      memory: 200Mi
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  restic:
+    tolerations: []
+    resources:
+      limits:
+        cpu: 200m
+        memory: 200Mi
+      requests:
+        cpu: 100m
+        memory: 100Mi
+## Configuration for cert-manager issuers.
+issuers:
+  ## Deploy let's encrypt ACME issuers
+  ## "letsencrypt-prod" and "letsencrypt-staging".
+  letsencrypt:
+    enabled: true
+    ## Namespaces in which to install the letsencrypt issuers in.
+    namespaces:
+      - kube-system
+      - monitoring
+    prod:
+      ## Mail through which letsencrypt can contact you.
+      email: letsencrypt@elastisys.com
+    staging:
+      ## Mail through which letsencrypt can contact you.
+      email: letsencrypt@elastisys.com
+  ## Additional issuers to create.
+  ## ref: https://cert-manager.io/docs/configuration/
+  extraIssuers: []
+  # - apiVersion: cert-manager.io/v1
+  #   kind: Issuer
+  #   metadata:
+  #     name: selfsigned-issuer
+  #     namespace: sandbox
+  #   spec:
+  #     selfSigned: {}
+## Configration for cert-manager and it's components.
+certmanager:
+  resources: {}
+  nodeSelector: {}
+  tolerations: {}
+  affinity: {}
+  webhook:
+    resources: {}
+    nodeSelector: {}
+    tolerations: {}
+    affinity: {}
+  cainjector:
+    resources: {}
+    nodeSelector: {}
+    tolerations: {}
+    affinity: {}
+metricsServer:
+  enabled: true
```

Same config change here.
```suggestion
  enabled: true
calicoAccountant:
  enabled: true
```